src/Resources/contao/dca/tl_member.php

```diff
@@ -617,42 +612,6 @@ public function storeDateAdded($dc)
 	}
 
 
-	/**
-	 * Check whether the user session should be removed
-	 *
-	 * @param DataContainer $dc
-	 */
-	public function checkRemoveSession($dc)
```

Comment:
No BC break if you remove a public method?

Reply:
It is a BC break for sure. The question is whether out BC promise includes DCA files, which I think they don't.